src/jit/jitconfig.cpp

```diff
@@ -45,7 +45,7 @@ void JitConfigValues::MethodSet::initialize(const wchar_t* list, ICorJitHost* ho
     {
         // Failed to convert the list. Free the memory and ignore the list.
         host->freeMemory(reinterpret_cast<void*>(const_cast<char*>(m_list)));
-        m_list = "";
+        m_list = nullptr;
```

Comment:
These failure cases are probably rare, but is downstream code prepared for `m_list` to be null?

If not you might add a new field to indicate that `m_list` should not be freed, and set that here and keep the value as the empty string, then check the new field in the destructor.